scripts/postInstall.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Go to project root
+ROOT_DIR=$(dirname "$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)")
+cd "$ROOT_DIR" || exit 1
+
+# Run patch-package
+npx patch-package
+
+# Install node_modules in subpackages, unless we're in a CI/CD environment,
+# where the node_modules for subpackages are cached separately.
+# See `.github/actions/composite/setupNode/action.yml` for more context.
+if [[ -n ${CI+x} ]]; then
```

Comment:
Source: https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash